pkg/volume/vsphere_volume/vsphere_volume_util_unsupported.go

```diff
@@ -0,0 +1,25 @@
+// +build !linux,!windows
+
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vsphere_volume
+
+import "errors"
+
+func verifyDevicePath(path string) (string, error) {
```

Comment:
I see build for this file  is `// +build !linux,!windows`, what are other supported build platform?

Reply:
There aren't any other supported platforms, that's why this file is called `_unsupported.go`. This is a common pattern in Go when introducing platform-specific files, to include an `_unsupported` file that will still allow things to compile on others but leaves it up to someone who wants to make it work to add the necessary code.